tensorboard/plugins/debugger/README.md

```diff
@@ -0,0 +1,175 @@
+# The Debugger Dashboard
+
+The debugger dashboard offers a graphical user interface for the [TensorFlow debugger](https://www.tensorflow.org/programmers_guide/debugger). For instance, this dashboard enables users to
+
+* pause and resume execution at specified nodes or numbers of steps.
+* visualize values of tensors over time.
+* associate tensors with specific lines in python code.
+
+This dashboard is in its **alpha release**. Some features are not yet fully functional.
+
+![The Debugger Dashboard](images/debugger_intro.png)
+
+# Setup
+
+## Start TensorBoard with the `debugger_port` flag.
+
+To enable the debugger dashboard, pass the `debugger_port` flag to TensorBoard. TensorBoard will then both receive gRPC messages from and issue gRPC messages to model logic via this port.
```

Comment:
Fill this paragraph please. 72 chars is good. 80 is ok. Note we have a markdown auto formatter internally you can probably run.